packages/tasit-sdk/src/TasitSdk.test.js

```diff
@@ -203,21 +207,28 @@ describe("Decentraland", () => {
       );
 
       it("should list marketplace estates sell orders (without wallet)", async () => {
-        const estates = [];
-        for (let id of estateIds) {
-          estates.push({
-            id,
-            name: await estate.getMetadata(id),
-          });
+        const orders = [];
+        const totalSupply = await estate.totalSupply();
+
+        const allEstatesIds = Array.from(
+          { length: totalSupply.toNumber() },
+          (v, k) => k + 1
+        );
+
+        for (let id of allEstatesIds) {
+          const order = await getEstateSellOrder(marketplace, estate, id);
+          orders.push(order);
         }
 
-        expect(estates[0].name).to.equal("cool estate 0x1");
+        expect(orders[0].price.toString()).to.equal(ONE.toString());
+        expect(orders[0].estateName).to.equal("cool estate 0x1");
```

Comment:
Maybe also an expectation that each of the orders is non-null or something? To show that it can successfully "list multilpe marketplace estates sell orders"?

Reply:
This is done now. Resolving.